From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@mwinkle__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mwinkle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

yushwang committed:
VPN device script

yushwang committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into devicescript

yushwang committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into devicescript

Commit:
rename 3 categories

carolinacmoravia committed:
C11447: LOCBLOCKER: Unloc string due to three extra character

Hello, @tamram,

Localization team has reported source content issue that causes issue for localization.

Please, help to check my comment tag added into the article and reply with explanation if source text should be fixed, or it is by design. If fix is needed, please, let me know either if you would like me to fix it in this PR, or if you will fix it in another PR. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

carolinacmoravia committed:
C11481:  Inconsistent format due to missing breakline

Hello, @daveba, 

Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

Lisaco88 committed:
edit pass: csharp-tutorial

GitHubber17 committed:
edit pass: active-directory-licensing-group-product-migration

ktoliver committed:
edit pass: extensions-dsc-overview.md and extensions-dsc-template.md

ktoliver committed:
fix link

ktoliver committed:
fix link

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into custom-vision-service-tutorials

Lisaco88 committed:
edit pass: python-tutorial

GitHubber17 committed:
edit pass: adjustments

yushwang committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into devicescript

carolinacmoravia committed:
Revert "M22773: Misleading sentence"

carolinacmoravia committed:
Update active-directory-saas-adpfederatedsso-tutorial.md

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into custom-vision-service-tutorials

Lisaco88 committed:
edit pass: custom-vision-service-tutorials

MGoedtel committed:
revised article to improve readability and details

MGoedtel committed:
updated to improve readability

MGoedtel committed:
updated to improve readability

MGoedtel committed:
updated to improve readability

Lisaco88 committed:
Deleted outdated line per author

GitHubber17 committed:
edit pass: machine-learning--scenario--predictive-maintenance

GitHubber17 committed:
remove locale specifier

GitHubber17 committed:
edits

v-geberr committed:
edits

ninarn committed:
Update query editor docs and images according to recent naming update.

yushwang committed:
Added more diagrams and completed the doc

yushwang committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into devicescript

ninarn committed:
Merge remote-tracking branch 'MicrosoftDocs/master'

yushwang committed:
TOC update

Commit:
Updated author metadata

jeffgilb committed:
Merge branch 'master' into jeff2211

Commit:
Updated TOC links

Commit:
updated TOC links

Commit:
Updated TOC links

yushwang committed:
Fixed typos & Acrolinx suggestions

Commit:
Updated troubleshooting article for acrolinx issues

yushwang committed:
More Acrolinx fixes

yushwang committed:
Fixed formatting issue with diagram

GitHubber17 committed:
edits

banisadr committed:
adding service bus updates to documentation

yushwang committed:
Screenshots touch ups

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

nitinme committed:
freshness updates

j-martens committed:
Quickstart update based on usability results

j-martens committed:
Fixing some formatting issues after the edits.

ajaycode committed:
Topology map

Rajani-Janaki-Ram committed:
editing dates

v-nagta committed:
Index Page Missing Icons Update

tjvishnu committed:
Updated Preview Boundaries

tjvishnu committed:
Edited minor errors

tjvishnu committed:
Updated Geos where the feature is available

Also added an appropriate hyperlink

j-martens committed:
Improving the history run instructions.

ktoliver committed:
edit pass: Python.md

ktoliver committed:
more edits

j-martens committed:
Formatting of Workbench quickstart

ktoliver committed:
more edits

j-martens committed:
Adding CLI information to Quickstart

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

j-martens committed:
Quickstart Acrolinx improvements

rayne-wiselman committed:
fix png name

rayne-wiselman committed:
fixing dup entry in TOC

rayne-wiselman committed:
Update hyper-v-vmm-network-mapping.md

bandersmsft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mmacy committed:
[aci] dns + cli version update

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

bandersmsft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

tjvishnu committed:
Updated minor errors and style

v-shils committed:
Merge pull request #33750 from ninarn/master

Updating query editor docs and images

Commit:
Merge pull request #33834 from nitinme/nitinme-freshness-updates

Freshness updates # 2

msangapu-msft committed:
Story 1185034 - Rename TOC item to App Service Linux

v-shils committed:
Merge pull request #33781 from j-martens/master

Quickstart update based on usability results

msangapu-msft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into LinuxToc

yushwang committed:
Formatting changes to address feedback

Changed the numbered items to bulleted list (non-sequential items)
Moved inline diagrams

msangapu-msft committed:
added "on"

PRMerger15 committed:
Merge pull request #33831 from mmacy/aci-dns-minor-fix

[aci] dns + cli version update

v-shils committed:
Merge pull request #33828 from rayne-wiselman/rayne-february22

Rayne february22

CarlRabeler committed:
Update sql-database-elastic-pool.md

Commit:
Responded to reviewer feedback

tamram committed:
metadata updates for storage articles

Brenduns committed:
Fixes for three broken articles - restoring accuracy

tdykstra committed:
fix some links

tdykstra committed:
dev --> master

MGoedtel committed:
Update article to address items

ktoliver committed:
fix merge conflict

MGoedtel committed:
updated script

tamram committed:
metadata updates for storage articles

ShawnJackson committed:
Update python-tutorial.md

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

ShawnJackson committed:
Merge pull request #33353 from GitHubber17/vsts-1181229

edit pass: active-directory-licensing-group-product-migration

ShawnJackson committed:
Update extensions-dsc-overview.md

ShawnJackson committed:
Update extensions-dsc-template.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
reporting01

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

nitinme committed:
addressing customer feedback

Ja-Dunn committed:
Merge pull request #32632 from v-savila/1174717

rename 3 categories

banisadr committed:
updating sb article with descriptions of data payload items

zhiweiwangmsft committed:
Update server datafreshness alert notice

Update server datafreshness alert notice

zhiweiwangmsft committed:
update the time span

update the time span

zhiweiwangmsft committed:
update grammar

grammer

billmath committed:
updating sso

billmath committed:
updating

Commit:
caintro02

GitHubber17 committed:
Merge pull request #33713 from MGoedtel/updateCrossQuery221

revised article to improve readability and details

GitHubber17 committed:
Update scenario-deep-learning-for-predictive-maintenance.md

GitHubber17 committed:
Update scenario-predictive-maintenance.md

Commit:
proxy02

zhiweiwangmsft committed:
remove typo

remove typo

zhiweiwangmsft committed:
fix bad link

update bad link

ajaycode committed:
added media directory - monitoring-network

Rajani-Janaki-Ram committed:
dates

Rajani-Janaki-Ram committed:
dates

msvijayn committed:
Log Alert doc update and corrections

msvijayn committed:
Final

msvijayn committed:
warning removed

msvijayn committed:
warning removal

tjvishnu committed:
Updated Availability geos

mayanknayar committed:
Azure Traffic Manager with Azure Site Recovery

New article on using Azure Traffic Manager with Azure Site Recovery

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into aadc

billmath committed:
updating date

Commit:
Updated next steps.

Commit:
Adjusted next steps.

mayanknayar committed:
Updating ATM with ASR doc

Error corrections and language improvements

mayanknayar committed:
Image changes for ATM with ASR article

Border correction for images

PRMerger20 committed:
Merge pull request #32650 from carolinacmoravia/patch-594

C11447: LOCBLOCKER: Unloc string due to three extra character

MGoedtel committed:
fixed link

EricRadzikowskiMSFT committed:
Change tag to note

EricRadzikowskiMSFT committed:
Change GA tag to note

v-geberr committed:
user voice on toc

itechedit committed:
Merge pull request #33801 from Rajani-Janaki-Ram/master

editing dates

PRMerger20 committed:
Merge pull request #33901 from EricRadzikowskiMSFT/patch-43

Change tag to note

bsiva committed:
Hyper-V policy copy frequency update

itechedit committed:
Merge pull request #33804 from v-nagta/missingiconsv2

Index Page Missing Icons Update

cherylmc committed:
Update log-analytics-network-performance-monitor-service-endpoint.md

PRMerger19 committed:
Merge pull request #33848 from MGoedtel/UpdateWinAgent222

Update article to address items

itechedit committed:
Merge pull request #33835 from msangapu/LinuxToc

minor App Service TOC update

itechedit committed:
Merge pull request #33869 from MarkusVi/reporting01

reporting01

itechedit committed:
Update active-directory-aadconnect-sso-quick-start.md

PRMerger16 committed:
Merge pull request #33903 from EricRadzikowskiMSFT/patch-47

Change GA tag to note

PRMerger12 committed:
Merge pull request #33884 from MarkusVi/proxy02

proxy02

v-geberr committed:
one word

PRMerger20 committed:
Merge pull request #33549 from carolinacmoravia/revert-32905-patch-602

Revert "M22773: Misleading sentence"

PRMerger12 committed:
Merge pull request #33905 from bsiva/master

Hyper-V policy copy frequency update

PRMerger10 committed:
Merge pull request #33749 from v-geberr/0221-data-extraction

edits

PRMerger20 committed:
Merge pull request #33906 from cherylmc/patch-1

change manager to monitor

itechedit committed:
Merge pull request #33751 from yushwang/devicescript

Download VPN devicescript

itechedit committed:
Merge pull request #33805 from tjvishnu/patch-1

Updated Public Preview Documentation for Azure File Shares Backup

PRMerger19 committed:
Merge pull request #33870 from nitinme/nitinme-freshness-updates

Addressing customer feedback

Commit:
Updating dates and explaining commands tag in manifest

PRMerger19 committed:
Merge pull request #33883 from MarkusVi/caintro02

caintro02

itechedit committed:
Merge pull request #33845 from Brenduns/jeff-updates

Fixes for three broken articles - restoring accuracy

Commit:
Updates.

Commit:
Updates.

itechedit committed:
Merge pull request #33880 from billmath/aadc

Aadc

PRMerger20 committed:
Merge pull request #33847 from tdykstra/patch-15

fix some links

PRMerger14 committed:
Merge pull request #32928 from carolinacmoravia/patch-606

C11481:  Inconsistent format due to missing breakline

Commit:
Fixes.

Commit:
Fix.

Commit:
Fix.

itechedit committed:
Merge pull request #33768 from banisadr/eg-add-sb-events

adding service bus updates to documentation

itechedit committed:
Merge pull request #33892 from ajaycode/master

Added media directory for network-monitoring

v-geberr committed:
changes based on Amir's response to ICM incident

PRMerger13 committed:
Merge pull request #33898 from jimdial/patch-77

Updated next steps.

itechedit committed:
Merge pull request #33844 from tamram/tamram-0222

metadata updates for storage articles

PRMerger14 committed:
Merge pull request #33899 from jimdial/patch-78

Adjusted next steps.

ShawnJackson committed:
Merge pull request #33160 from Lisaco88/custom-vision-service-tutorials

edit pass: csharp-tutorial

PRMerger12 committed:
Merge pull request #33839 from CarlRabeler/patch-5

Update sql-database-elastic-pool.md

ShawnJackson committed:
Merge pull request #33357 from ktoliver/1182101-b

edit pass: extensions-dsc-overview.md and extensions-dsc-template.md

PRMerger17 committed:
Merge pull request #33877 from zhiweiwangmsft/patch-9

Update server datafreshness alert notice

ktoliver committed:
Merge pull request #33904 from v-geberr/0223-uservoice-in-toc

user voice on toc

ShawnJackson committed:
Merge pull request #33744 from GitHubber17/vsts1186495

edit pass: machine-learning--scenario--predictive-maintenance

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

GitHubber17 committed:
Merge pull request #33896 from mayanknayar/master

Azure Traffic Manager with Azure Site Recovery

ShawnJackson committed:
Merge pull request #33809 from ktoliver/python

edit pass: cognitive-services/computer-vision/quickstarts/python.md

v-alje committed:
Merge pull request #33873 from MicrosoftDocs/FromPublicRepo

From public repo

bandersmsft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

itechedit committed:
Merge pull request #33893 from msvijayn/connect-vijayn

Log Alert doc update and corrections

bandersmsft committed:
cost mgt - updated ea register

PRMerger20 committed:
Merge pull request #33908 from v-geberr/0223-concept-entity

changes based on Amir's response to ICM incident

GitHubber17 committed:
Merge pull request #33752 from jeffgilb/jeff2211

Updated author metadata

GitHubber17 committed:
Merge pull request #33907 from mani-ramaswamy/master

Updating dates and explaining commands tag in manifest

Commit:
Merge pull request #33911 from bandersmsft/master

cost mgt - updated ea register
